src/client/fsharp/ASTViewer/State.fs

```diff
@@ -2,8 +2,10 @@ module FantomasTools.Client.ASTViewer.State
 
 open System
 open Elmish
+open FantomasTools.Client.Editor
```

Comment:
Pedantic nitpick, but how about moving this `open` down to keep them sorted?

Reply:
Good spot!